frontend/src/pages/BookGalleryPage/bookGalleryPage/BookGalleryPage.tsx

```diff
@@ -1,13 +1,16 @@
 import {Book} from "../../../types/types.ts";
 import BookGallery from "../components/bookGallery/BookGallery.tsx";
 import "./BookGalleryPage.css";
+import AddBookButton from "../components/addBookButton/AddBookButton.tsx";
+import {Link} from "react-router-dom";
 
 type BookGalleryPageProps = {
     data: Book[]
 }
 export default function BookGalleryPage({data}: BookGalleryPageProps) {
     return (
         <>
+            <Link to={"/books/add"}><AddBookButton /></Link>
```

Comment:
I think it's better to either use a <Link> (which can be styled like a button) OR a <button> with the useNavigate